sbnanaobj/StandardRecord/SRCRUMBSResult.h

```diff
@@ -0,0 +1,42 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRCRUMBSResult.h
+////////////////////////////////////////////////////////////////////////
+#ifndef SRCRUMBSRESULT_H
+#define SRCRUMBSRESULT_H
+
+
+#include <vector>
+
+namespace caf {
+/// Representation of CRUMBS MVA Slice ID output
+class SRCRUMBSResult {
+  public:
+  SRCRUMBSResult();
+  ~SRCRUMBSResult() {}
+
+  float score;
+  float tpc_CRFracHitsInLongestTrack;
+  float tpc_CRLongestTrackDeflection;
+  float tpc_CRLongestTrackDirY;
+  float tpc_CRNHitsMax;
+  float tpc_NuEigenRatioInSphere;
+  float tpc_NuNFinalStatePfos;
+  float tpc_NuNHitsTotal;
+  float tpc_NuNSpacePointsInSphere;
+  float tpc_NuVertexY;
+  float tpc_NuWeightedDirZ;
+  float tpc_StoppingChi2CosmicRatio;
+  float pds_FMTotalScore;
+  float pds_FMPE;
+  float pds_FMTime;
+  float crt_TrackScore;
+  float crt_HitScore;
+  float crt_TrackTime;
+  float crt_HitTime;
```

Comment:
<broken record>

Can we make these names more CAF-y?

I would actually be a fan of making three smaller classes so these can be `tpc.`, `pds.`, `crt.`. Some of these would work out well in CAF style, e.g. `crt.tracktime`. Some of the longer ones might be a struggle.

Reply:
Really liked the idea of separate objects, not so sure about all the lower case names for the tpc vars but have gone with it to match convention.